source/constructs/api/discovery_job/crud.py

```diff
@@ -104,14 +104,14 @@ def get_running_run(job_id: int) -> models.DiscoveryJobRun:
     return db_run
 
 
-def __add_job_databases(run: models.DiscoveryJobRun, database_type: DatabaseType, base_time_dict: dict):
-    databases = get_catalog_database_level_classification_by_type_all(database_type.value).all()
+def __add_job_databases(run: models.DiscoveryJobRun, database_type: str, base_time_dict: dict):
+    databases = get_catalog_database_level_classification_by_type_all(database_type).all()
     for database in databases:
-        base_time = base_time_dict.get(f'{database.account_id}-{database.region}-{database_type.value}-{database.database_name}')
+        base_time = base_time_dict.get(f'{database.account_id}-{database.region}-{database_type}-{database.database_name}')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You are using the `get` method without a default argument to return the value of a key in a dictionary. We recommended that you use a default argument so that if the value for your key is not found, a default value is returned. If a default value is not provided and the key is not found, then `None` is returned.

[Learn more](https://docs.python.org/3/library/stdtypes.html#dict.get)